packages/bitcore-client/bin/wallet-paypro

```diff
@@ -67,9 +67,15 @@ const main = async () => {
   if (!['BTC', 'BCH', 'DOGE', 'LTC'].includes(chain)) {
     nonce = inputNonce || await wallet.getNonce();
   }
-  const currency = token || chain;
+
+  const { data: currencies } = await doRequest({
+    method: 'GET',
+    uri: 'https://bitpay.com/currencies',
```

Comment:
Shouldn't it get everything it needs from the paypro response?

Reply:
I'll check, but I don't think the PP response returns currency metadata. This call is to get the native currency for L2 chains (e.g. the OP chain's native currency is ETH_op...which either needs to be hard coded somewhere or I can pull it programmatically from this API call)

Reply:
Turns out, it's trivial to add the needed metadata to the PP options response. Also, the below logic to find the native currency can be circumvented with --token until the PP options includes the native metadata.

I've removed this API call.